djconnectwise/models.py

```diff
@@ -1487,9 +1510,9 @@ class Ticket(UpdateConnectWiseMixin, TimeStampedModel):
         'project': 'project',
         'phase': 'phase'
     })
+    # TODO do we actually need this? Sources and contacts can be on project
+    #  tickets
```

Comment:
@kti-cameron This was my last question, I had forgotten about this one. Why did we originally not have sources/contacts on project tickets? Was it just because we didn't plan on editing them at the time so they were separated? Or is there a different reason? It looks like its fine to change those in the PSA.